fio/mmap_io.go

```diff
@@ -0,0 +1,105 @@
+package fio
+
+import (
+	"errors"
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+// DefaultMemMapSize 最大映射内存大小
+const DefaultMemMapSize = 256 * 1024 * 1024
+
+// MMapIO 标准系统文件IO
+type MMapIO struct {
+	fd     *os.File // 系统文件描述符
+	data   []byte   // 与文件对应的映射区
+	dirty  bool     // 是否更改过
+	offset int64    // 写入位置
+}
+
+// NewMMapIOManager 初始化标准文件 IO
+func NewMMapIOManager(fileName string) (*MMapIO, error) {
```

Comment:
![image](https://github.com/ByteStorage/FlyDB/assets/88550136/5cd3250b-94c9-41ac-a8c2-33a18a7bbcc7)
you can replace it with this and test the other `db` tests can run or not

Reply:
and show the benchmark test result
![image](https://github.com/ByteStorage/FlyDB/assets/88550136/eae777a9-883c-4813-8a88-ec3a3d512957)

Reply:
the benchmark test is pass, 1879 ns/op(mmap IO) VS 3181 ns/op(file IO)
![C29`~AVA{4T3PGJNI7GN4SQ](https://github.com/ByteStorage/FlyDB/assets/102655073/05946381-3fb6-4742-9122-d07d6b0949e5)

![NC $7{G0{1WX~LK4S12( C1](https://github.com/ByteStorage/FlyDB/assets/102655073/e5ef32dd-e1da-4816-8e57-3230e6656fa4)

Reply:
Benchmark TestPutAndGet result（50w operations）

![7dd1bbe4ef8581af51ce0d4fcb834b9](https://github.com/ByteStorage/FlyDB/assets/102655073/d37a0459-d731-47f8-98df-7913f4b38c8c)